server/player.lua

```diff
@@ -835,11 +835,11 @@ function CreatePlayer(playerData, Offline)
     ---@deprecated use ox_inventory exports directly
     ---@param item string
     ---@param amount number
-    ---@param slot? number
     ---@param metadata? table
+    ---@param slot? number
     ---@return boolean success
-    function self.Functions.AddItem(item, amount, slot, metadata)
-        return exports.ox_inventory:AddItem(self.PlayerData.source, item, amount, slot, metadata)
+    function self.Functions.AddItem(item, amount, metadata, slot)
```

Comment:
```suggestion
    function self.Functions.AddItem(item, amount, slot, metadata)
```
QBCore has them in this order. Reference: [here](https://docs.qbcore.org/qbcore-documentation/qb-core/player-data#qbcore.player.createplayer:~:text=self.Functions.AddItem%20%3D%20function(item%2C%20amount%2C%20slot%2C%20info))